development/build/scripts.js

```diff
@@ -721,57 +793,47 @@ function setupSourcemaps(buildConfiguration, { devMode }) {
   });
 }
 
-async function bundleIt(buildConfiguration) {
-  const { label, bundlerOpts, events } = buildConfiguration;
-  const bundler = browserify(bundlerOpts);
-
-  // manually apply non-standard options
-  bundler.external(bundlerOpts.manualExternal);
-  bundler.ignore(bundlerOpts.manualIgnore);
-  if (Array.isArray(bundlerOpts.manualExclude)) {
-    bundler.exclude(bundlerOpts.manualExclude);
-  }
-
+async function bundleIt({ label, buildBundler, events }) {
+  const bundler = buildBundler();
   // output build logs to terminal
   bundler.on('log', log);
-
   // forward update event (used by watchify)
-  bundler.on('update', () => performBundle());
+  bundler.on('update', () => performBundle({ bundler, events }));
 
   console.log(`Bundle start: "${label}"`);
-  await performBundle();
+  await performBundle({ bundler, events });
   console.log(`Bundle end: "${label}"`);
+}
 
-  async function performBundle() {
-    // this pipeline is created for every bundle
-    // the labels are all the steps you can hook into
-    const pipeline = labeledStreamSplicer([
-      'groups',
-      [],
-      'vinyl',
-      [],
-      'sourcemaps:init',
-      [],
-      'minify',
-      [],
-      'sourcemaps:write',
-      [],
-      'dest',
-      [],
-    ]);
-    const bundleStream = bundler.bundle();
-    // trigger build pipeline instrumentations
-    events.emit('configurePipeline', { pipeline, bundleStream });
-    // start bundle, send into pipeline
-    bundleStream.pipe(pipeline);
-    // nothing will consume pipeline, so let it flow
-    pipeline.resume();
-
-    await endOfStream(pipeline);
-
-    // call the completion event to handle any post-processing
-    events.emit('bundleDone');
-  }
+async function performBundle({ bundler, events }) {
```

Comment:
I didn't see a real reason this was nested within `bundleIt`, and considering most of `bundleIt` is this function anyway, I pulled it out.